packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -68,7 +70,7 @@ class DevToolsAppState extends State<DevToolsApp> {
               EmptyScreen.timeline,
               const PerformanceScreen(),
               EmptyScreen.memory,
-              EmptyScreen.debugger,
+              if (kIsWeb) const DebuggerScreen(),
```

Comment:
Modify the EmptyScreen.debugger to explain that it is not supported on Flutter Desktop.
That is important to keep the app display size consistent on Web and Desktop and make it obvious to users why certain things don't work on desktop.
Otherwise we get "there is a bug and I can't open the debugger screen on desktop".

Reply:
I've added in the warning message.

Note that we cannot guarantee a consistent app display size.  Someone can always turn on a system accessibility setting for larger text, or window scaling, or font changes, or plug in a higher dp monitor.  All will change the layout.